hedera-node/src/main/java/com/hedera/services/exceptions/ValidationUtils.java

```diff
@@ -0,0 +1,54 @@
+package com.hedera.services.exceptions;
+
+/*-
+ * ‌
+ * Hedera Services Node
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import com.hederahashgraph.api.proto.java.ResponseCodeEnum;
+
+import java.util.function.Supplier;
+
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.FAIL_INVALID;
+
+/**
+ * A minimalist collection of helpers to improve readability of code
+ * that throws an {@code InvalidTransactionException}.
+ *
+ * The {@link ValidationUtils#checkInvariant(boolean, Supplier)} helper
+ * should be used to assert system invariants hold.
+ */
+public class ValidationUtils {
+	public static void validateTrue(boolean flag, ResponseCodeEnum code) {
+		if (!flag) {
+			throw new InvalidTransactionException(code);
+		}
+	}
+
+	public static void validateFalse(boolean flag, ResponseCodeEnum code) {
+		if (flag) {
+			throw new InvalidTransactionException(code);
+		}
+	}
+
+	public static void checkInvariant(boolean flag, Supplier<String> failureMsg) {
```

Comment:
Is this just `checkTrue` but with an overloaded set of args? I'm wondering if maybe we should have:
```
public static void validateTrue(boolean flag, ResponseCodeEnum code) { ... }
public static void validateTrue(boolean flag, ResponseCodeEnum code, Supplier<String> failureMsg) { ... }
public static void validateFalse(boolean flag, ResponseCodeEnum code) { ... }
public static void validateFalse(boolean flag, ResponseCodeEnum code, Supplier<String> failureMsg) { ... }
```

Reply:
Removed `checkInvariant` and used the above signatures.